### PR TITLE
Update etcd-operator to latest release

### PR DESCRIPTION
**Pull request by lachie83**
bump etcd and etcd-operator version and update notes

### CODE REVIEW

**Comment by lachie83** [comment]
cc @chancez

File: stable/etcd-operator/templates/NOTES.txt [hunk 1]
```diff
@@ -8,12 +8,12 @@ Not enabling cluster, the ThirdPartResource must be installed before you can cre
 1. Watch etcd cluster start
   kubectl get pods -l etcd_cluster={{ .Values.cluster.name }} --namespace {{ .Release.Namespace }} -w
 2. Confirm etcd cluster is healthy
-  $ kubectl run --rm -i --tty --env="ETCDCTL_API=3" --env="ETCDCTL_ENDPOINTS=http://{{ .Values.cluster.name }}-0000:2379" etcd-test --image quay.io/coreos/etcd --restart=Never -- /bin/sh -c 'watch -n1 "etcdctl  member list"'
+  $ kubectl run --rm -i --tty --env="ETCDCTL_API=3" --env="ETCDCTL_ENDPOINTS=http://{{ .Values.cluster.name }}-client:2379" etcd-test --image quay.io/coreos/etcd --restart=Never -- /bin/sh -c 'watch -n1 "etcdctl  member list"'
```
Comment (chancez) [inline stable/etcd-operator/templates/NOTES.txt:11]: I didn't think etcd-operator deployed a service by default. Are you sure this is correct?

Comment (lachie83) [reply stable/etcd-operator/templates/NOTES.txt:11]: Looks like a change in behavior. This never used to be the case as you mentioned. Deploy this update and take a look.

Comment (rimusz) [reply stable/etcd-operator/templates/NOTES.txt:11]: LGTM

**Comment by lachie83** [comment]
@chancez have you had a chance to run this up?

**Comment by chancez** [comment]
@lachie83 I haven't, I'm fairly busy atm and the cluster I've been using for TPR stuff is having some troubles, but overall the changes look reasonable if the service' name is `${clustername}-client`.

**Comment by lachie83** [comment]
@rimusz @chancez push another commit to change this to the latest (now 0.3.2) and also include conditional creation of rbac roles/rolebindings for clusters with rbac enabled. I've tested this on a 1.6 cluster with rbac enabled.

**Comment by rimusz** [comment]
LGTM

**Comment by lachie83** [comment]
@prydonius @rimusz can you take another pass at this?

**Pull request merged**